atef/annotated_plans.py

```diff
@@ -0,0 +1,615 @@
+"""
+Plans with additional annotations to facilitate gui auto-generation
+In the future this should probably be pulled out into a separate repository
+as it will probably see considerable use
+"""
+from typing import (Any, Callable, Dict, Generator, Iterable, List, Optional,
+                    Tuple, Union)
+
+import bluesky.plans as bp
+from bluesky_queueserver import parameter_annotation_decorator
+
+
+def add_docstring_from(func):
+    def wrapper(orig_func):
+        if not orig_func.__doc__:
+            orig_func.__doc__ = ""
+        orig_func.__doc__ += (func.__doc__ or "")
+        return orig_func
+
+    return wrapper
+
+
+@parameter_annotation_decorator({
+    "parameters": {
+        "detectors": {"annotation": "typing.List[__DEVICE__]"},
+        "num": {"annotation": "int"},
+        "delay": {"annotation": "typing.Union[typing.Iterable[float], float]"},
+    }
+})
+@add_docstring_from(bp.count)
+def count(
+    detectors: List[Any],
```

Comment:
Could we do `Any` -> `ophyd.Device` since the decorator overwrites the annotation for the queue server's purposes?

Reply:
An upcoming[ bluesksy queueserver PR](https://github.com/bluesky/bluesky-queueserver/pull/288) should let us do better than `Any`, though I think we could use `ophyd.Device` here since the decorate overrides